consensus/polybft/block_builder.go

```diff
@@ -194,17 +195,8 @@ func (b *BlockBuilder) writeTxPoolTransaction(tx *types.Transaction) (bool, erro
 		return true, nil
 	}
 
-	if tx.ExceedsBlockGasLimit(b.params.GasLimit) {
```

Comment:
This probably should be reverted to the old code. This way we lose `b.params.TxPool.Drop(tx)` if tx could not be written to the block

Reply:
I think this is good, because, the same workflow is preserved. If an error happens in `WriteTxn` it will be handled in this function. As you see, the bottom else inside `if err != nil` block, drops the transaction.